cmd/monitor/import.go

```diff
@@ -24,36 +24,42 @@ import (
 	dd "github.com/tani-yu/dogleash/datadog"
 
 	"github.com/spf13/cobra"
-	"gopkg.in/zorkian/go-datadog-api.v2"
+	datadog "gopkg.in/zorkian/go-datadog-api.v2"
 )
 
-var inputPath string
+var target string
 
 // compute_checkCmd represents the compute_check command
 var monitorImportCmd = &cobra.Command{
 	Use:   "import",
-	Short: "json形式のファイルを読み込んでdatadogにdashboardを作成します",
+	Short: "Read json format file and create monitoring item on datadog",
 	Run: func(cmd *cobra.Command, args []string) {
 		cli, err := dd.NewDDClient()
 		if err != nil {
-			log.Fatalf("fatal: %s\n", err)
+			log.Fatalf("Failed to connect Datadog API server: %s\n", err)
 		}
 
-		raw, err := ioutil.ReadFile(inputPath)
-		if err != nil {
-			log.Fatalf("fatal: %s\n", err)
-			os.Exit(1)
+		if len(target) == 0 {
+			fmt.Println("Hello", target)
 		}
 
-		var monits []datadog.Monitor
-		json.Unmarshal(raw, &monits)
+		for _, inputPath := range args {
```

Comment:
In this large for loop, there are two processing; one is for parsing JSON objects from the files and the other is for posting unmarshaled JSON objects to Datadog API endpoint. IMO, you can separate these tasks into each loop. Let's say, at first, you should append unmarshaled monitor structures into one large slice of monitor structures and then post them to the endpoint in separate for-loop. It can reduce the depth of for-loop.